core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-filter.model.ts

```diff
@@ -23,23 +23,27 @@ export class ContributorAdminDashboardFilter {
   topicIds: string[];
   languageCode?: string;
   sort?: string | null;
+  firstActivity?: number;
   lastActivity?: number;
 
   /**
    * @param {String} languageCode - Language Code to filter for.
    * @param {String[]} topicIds - keywords to filter for.
    * @param {string} sort - sort options.
+   * @param {number} firstActivity - number of days since first activity.
    * @param {number} lastActivity - number of days since last activity.
    */
   constructor(
     topicIds: string[],
     languageCode?: string,
     sort?: string | null,
+    firstActivity?: number,
     lastActivity?: number
```

Comment:
Rename to `numberOfDaysSinceLastActivityStartDate` and `numberOfDaysSinceLastActivityEndDate`.

Reply:
done